src/tests/common/config_helper.hpp

```diff
@@ -28,7 +28,7 @@ inline SerenityConfig createAssuranceAnalyzerCfg(
 }
 
 inline SerenityConfig createThresholdDetectorCfg(
-  const double_t utilization = detector::DEFAULT_UTILIZATION_THRESHOLD) {
+    const double_t utilization = detector::DEFAULT_UTILIZATION_THRESHOLD) {
```

Comment:
This kind of tests are rather in units like `threshold_detector_test.cpp`, right? 
Then we should move them there.
Also: inline is unnecesary.